CMakeLists.txt

```diff
@@ -131,6 +131,11 @@ if(CMAKE_C_COMPILER_ID MATCHES "AppleClang|Clang|GNU|Intel")
   endif()
 endif()
 
+# Architecture flags
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm")
+    list(APPEND mi_cflags -march=native)
```

Comment:
I have no idea about cross compilation for Arm targets. Would it be affected because of this change?

Reply:
I did not get but updating for testing from host data.